WordPress/src/main/res/layout/debug_cookie_item.xml

```diff
@@ -48,7 +48,6 @@
         android:layout_height="wrap_content"
         android:background="?attr/selectableItemBackground"
         android:padding="@dimen/margin_medium"
-        android:src="@drawable/ic_delete_black_24dp"
```

Comment:
I couldn't find the drawable in the trunk branch, anybody know where the file is?

Reply:
Indeed this icon is missing but probably it doesn't create a crash because it is a debug only functionality. I think we could add an existing alternative icon like `ic_trash_grey_dark_24dp` to allow deleting a cookie (Me debug settings > overflow menu > debug cookies).

<img src="https://github.com/wordpress-mobile/WordPress-Android/assets/304044/7af14733-cc8c-4ca9-a30d-b0187ea0a6f9" width=360 />